manim/scene/vector_space_scene.py

```diff
@@ -79,20 +79,90 @@ def add_plane(self, animate: bool = False, **kwargs):
         self.add(plane)
         return plane
 
-    def add_axes(self, animate: bool = False, color: bool = WHITE, **kwargs):
+    def add_axes(
+        self,
+        animate: bool = False,
+        animation: Animation = Create,
```

Comment:
### animation, animation_kwargs and play_kwargs
Why must a user be limited to the `Create` animation? -> Solution: my changes
It would annoy me if I wanted to create a more advanced animation but I would have only `Create` animation available.